gulpfile.js

```diff
@@ -385,10 +385,6 @@ gulp.task('run', ['clean', 'serve'], function() {
   util.log('Done!');
 });
 
-// Watch for changes
-gulp.task('watch', ['build'], function() {
-  gulp.watch(config.folderAssets.base + '/**/*.scss', ['sass']);
-});
```

Comment:
You should remove the hole file, right?

Reply:
Not right now, only dist task is obsolete in this file, I'm doing this step by step, else it's a mess.

Reply:
What a deception! :P